src/StaticReflection/SourceLocator/ParentAttributeSourceLocator.php

```diff
@@ -61,7 +61,7 @@ public function locateIdentifier(Reflector $reflector, Identifier $identifier):
                 $classReflector,
                 $class,
                 $fakeLocatedSource,
-                new Namespace_(new Name('Symfony\Component\DependencyInjection\Attribute'))
+                'Symfony\Component\DependencyInjection\Attribute'
```

Comment:
@TomasVotruba this is due to new latest release PHPStan 1.8.7 which the `BetterReflection` scoped require `?string` instead of `Namespace_`, ref https://github.com/ondrejmirtes/BetterReflection/blob/ceba8a6f7b2feeb4d1d4caa8f8c146af777c6df6/src/Reflection/ReflectionClass.php#L283